canvas_modules/common-canvas/__tests__/common-properties/operators/dm-condition-operators-test.js

```diff
@@ -93,14 +93,15 @@ describe("dm condition operators work correctly", () => {
 		dropdownButton.simulate("click");
 		dropdownList.at(3).simulate("click"); // Fulfill Condition by selecting item with dmRole discrete
 		errorMessages = controller.getErrorMessages();
-		expect(errorMessages).to.deep.equal({});
+		// TODO: this works in the UI but errorMessages is not updated in test
```

Comment:
This test works in UI but `controller.getErrorMessages()` is not updated after `wrapper.update()`.